modules/aspect_bazel_lib/1.2.1/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+build_targets: &build_targets
+  - "@aspect_bazel_lib//lib/tests:expand_template_test"
```

Comment:
Do you actually want to run this test or just building it is enough?

Reply:
We had this in our previous presubmit files, but I'm not exactly sure what it's doing. Is it building all targets except for this one? Is there a spec for `presubmit.yml` I can take a look at somewhere?

Reply:
This means, the presubmit will only try to build `@aspect_bazel_lib//lib/tests:expand_template_test`, not any other targets and not actually run the test. If you change `build_targets` to `test_targets`, then the test will run after building it.

https://github.com/bazelbuild/bazel-central-registry#presubmityml maybe?

A good example would be https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/rules_jni/0.4.0/presubmit.yml

Reply:
Thanks @meteorcloudy. I updated `presubmit.yml` to exercise our bzlmod e2e. Does this look right?

Reply:
Nice, do you also want to specify build targets that you want to expose? For example, specifying `@rules_jni//jni/...` like this will make sure all targets under `@rules_jni//jni` are buildable.
https://github.com/bazelbuild/bazel-central-registry/blob/0bcf433b2dbc4ce96dfce3e7e7e0e1fbf140c9e2/modules/rules_jni/0.4.0/presubmit.yml#L5-L9